src/api/controllers/adminPermissionsController.js

```diff
@@ -0,0 +1,165 @@
+import models from '../models';
+import statusCodes from '../../helpers/constants/status.codes';
+import generatePagination from '../../helpers/generate.pagination.details';
+
+const {
+  Article, Report, User, Reason
+} = models;
+
+/**
+ *
+ *
+ * this is the article controller
+ *
+ * @export
+ * @class ArticleController
+ */
+
+/**
+ * containing all article's model controllers
+ * @export
+ * @class ArticleController
+ */
+export default class adminPermissionsController {
+  /**
+   * fetch all reported Articles
+   *
+   * @static
+   * @param {array} req - request array
+   * @param {array} res - respond array
+   * @returns {array} response body
+   * @memberof getArticles
+   */
+  static async getReportedArticles(req, res) {
```

Comment:
don't you think ``getAllReportedArticles`` will be better as a class method name

Reply:
I think that will be a long name which is not necessary because its ```articles``` not article,
@MCFrank16